e2e/resources/extensibility-configs/context-submenus-ext.json

```diff
@@ -262,18 +262,6 @@
       }
     ],
     "toolbar": [
-      {
-        "id": "app.toolbar.searchFilter",
-        "order": 50,
-        "title": "APP.BROWSE.SEARCH.TOGGLE_SEARCH_FILTER",
```

Comment:
Please check if this i18n key is not used anywhere else and remove from translations